patches/nx+16.8.1.patch

```diff
@@ -1,9 +1,9 @@
 diff --git a/node_modules/nx/src/command-line/yargs-utils/shared-options.js b/node_modules/nx/src/command-line/yargs-utils/shared-options.js
-index 0f92f37..5b625b1 100644
+index 857b809..1c18464 100644
 --- a/node_modules/nx/src/command-line/yargs-utils/shared-options.js
 +++ b/node_modules/nx/src/command-line/yargs-utils/shared-options.js
 @@ -150,6 +150,10 @@ function withRunManyOptions(yargs) {
-         describe: '[deprecated] Run the target on all projects in the workspace',
+         describe: '[deprecated] `run-many` runs all targets on all projects in the workspace if no projects are provided. This option is no longer required.',
```

Comment:
is this needed?

Reply:
We didn't make this change. It came with the version upgrade of the Nx package.